pkg/util/kubeobjects/deployment/deployment.go

```diff
@@ -55,11 +58,11 @@ func CreateOrUpdateDeployment(c client.Client, logger logr.Logger, desiredDeploy
 		return false, err
 	}
 
-	if !IsHashAnnotationDifferent(currentDeployment, desiredDeployment) {
+	if !hasher.IsHashAnnotationDifferent(currentDeployment, desiredDeployment) {
```

Comment:
```suggestion
	if !hasher.IsAnnotationDifferent(currentDeployment, desiredDeployment) {
```

Reply:
done in [542e38f](https://github.com/Dynatrace/dynatrace-operator/pull/2336/commits/542e38f34e74944a9b6e7e5db9623aa2d7969e7e)